pegasus/sites.v3/code.org/public/starwars.haml

```diff
@@ -43,8 +43,8 @@ social:
       %div.sw-tutorial-info
         %p.sw-tutorial-description= I18n.t(:starwars_blocks_description)
         %p.sw-tutorial-specs= I18n.t(:starwars_blocks_specs)
-      %a{:href => '#'}
-        %button.disabled.sw-disabled{disabled:"disabled"}= I18n.t(:starwars_blocks_start_button)
```

Comment:
Did you intend to change the string from `starwars_blocks_start_button` to `starwars_javascript_start_button`?

Also, you should probably delete the branch after you're PR has been merged. Thanks!